Sources/Valet/SecureEnclaveValet.swift

```diff
@@ -119,7 +119,7 @@ public final class SecureEnclaveValet: NSObject {
     ///   - object: A Data value to be inserted into the keychain.
     ///   - key: A key that can be used to retrieve the `object` from the keychain.
     /// - Throws: An error of type `KeychainError`.
-    /// - Important: Inserting data larger than 1mb in size may sporadically fail.
+    /// - Important: Inserted data should be smaller than 4kb.
```

Comment:
supernit: It looks like it supports _up to_ 4kb, rather than requiring it be smaller than that.
```suggestion
    /// - Important: Inserted data should be no larger than 4kb.
```